### ISSUE TITLE
[ feature request ] list repos for organization

### ISSUE THREAD

**Issue by yonas** [XAMPPRocky/octocrab#734]
List all forks of an organization owned by the currently authenticated user.

For example, in 

```rust
use octocrab::Octocrab;

#[tokio::main]
async fn main() -> octocrab::Result<()> {
    let token = std::env::var("GITHUB_TOKEN").expect("GITHUB_TOKEN env variable is required");

    let octocrab = Octocrab::builder().personal_token(token).build()?;

    let my_repos = octocrab
        .current()
        .list_repos_for_authenticated_user()
        .type_("owner")
        .sort("updated")
        .per_page(100)
        .send()
        .await?;

    for repo in my_repos {
        println!("{}", repo.name);
    }

    Ok(())
}
```

I would like to replace `list_repos_for_authenticated_user() ` with something like `list_repos_for_authenticated_user("myorg")`.

**Comment by dmgorsky** [comment]
There are APIs for 
"List organizations for the authenticated user"
https://docs.github.com/en/rest/orgs/orgs?apiVersion=2022-11-28#list-organizations-for-the-authenticated-user
and
"List organization repositories"
https://docs.github.com/en/rest/repos#list-organization-repositories

Will they be enough? I can check the code then.

**Comment by yonas** [comment]
@dmgorsky Thanks, but those are not sufficient.

I want to pass in the organization since I have multiple orgs.

**Comment by anoldguy** [comment]
I was curious about this as well. My use case involves listing orgs that a user is a member of (List Organizations for the Authenticated User 👍), and then listing repos the user has permissions for under that organization (List Organization Repositories 👍).

I'd offer a PR, but I'm using this lib as a part of my 'learn rust' project, and have hardly any rust experience at this point.



> I want to pass in the organization since I have multiple orgs.

@yonas, are you sure the second link there ([List Organization Repositories](https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#list-organization-repositories), modified for API version) isn't what you're looking for? It requires you to pass the org: `/orgs/{org}/repos`.

**Comment by yonas** [comment]
@anoldguy You're right, implementing [List Organization Repositories](https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#list-organization-repositories) would work for me.
